api/staticRoutes.js

```diff
@@ -0,0 +1,69 @@
+const Router = require('express').Router();
+
+const Log = require('./lib/log');
+const Plan = require('./model/plan');
+const config = require('../src/config.json');
+
+// all plan pages should render opengraph tags
+Router.get(['/plan/:planId', '/plan/:planId/*'], function(req, res, next) {
+  const { planId } = req.params;
+
+  // fetch plan details to populate opengraph tags
+  Plan.fetchByPlanID(planId)
+    .then(plan => {
+      let pageDescription;
+
+      // some plans don't have depositing dates
+      if (plan.attributes.data.DEPOSITING_DATE)
+        pageDescription = `תב"ע ב${plan.attributes.data.PLAN_COUNTY_NAME} אשר הופקדה ב ${dateStrFormat(plan.attributes.data.DEPOSITING_DATE)}`;
```

Comment:
The strings should be in the template.
You can have an if condition with mustache.